tests/cli.test.ts

```diff
@@ -33,7 +34,7 @@ const tests: encodeTest[] = [
   ],
   ['mint(bool,bool,bool,bool)', ['true', 'false', '1', '0'], ['0x1', '0x0', '0x1', '0x0']],
   ['mint(bytes8)', ['0xffffffffffffffff'], ['0xffffffffffffffff']],
-  ['byte_me(bytes)', [['0xff', '0xff', '0xff', '0xff']], ['0x4', '0xff', '0xff', '0xff', '0xff']],
+  ['byte_me(bytes[])', [['0xff', '0xff', '0xff', '0xff']], ['0x4', '0xff', '0xff', '0xff', '0xff']],
```

Comment:
are the cli tests run in CI?

Reply:
Yup, I realized of the encoding bugs because they were failing in CI